styles.css

```diff
@@ -0,0 +1,446 @@
+    content: "";
+    position: absolute;
+    border-top: 70px solid transparent;
+    border-left: 100vw solid white;
+    border-bottom: 0 solid transparent;
+    bottom: 0;
+    left: 0;
+}
+
+.header {
+    color: var(--lightgrey-color);
+    text-align: center;
+    margin-bottom: 50px;
+}
+
+.header-text {
+    padding: 50px 0;
+}
+
+header h1 {
+    padding: 40px 0;
+}
+
+.header-image {
+    position: absolute;
+    height: 4rem;
+    bottom: 20px;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+.intro {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    color: var(--purple-color);
+    text-align: center;
+    font-weight: 700;
+    font-size: 1.75rem;
+    margin-bottom: 50px;
+    font-family: 'Merriweather', serif;
+    position: relative;
+}
+
+.intro p {
+    padding: 10px;
+}
+
+.small-text {
+    font-size: 12px;
+}
+
+.intro:after {
+    content: "hello";
+    position: absolute;
+    color: var(--bg-words-color);
+    font-family: 'Roboto', sans-serif;
+    font-weight: 700;
+    font-size: 400px;
+    z-index: -1;
+    top: -150px;
+
+}
+
+
+/* --------------- 4.3 Work ---------------------------- */
+.case-studies-wrapper {
+    background-color: var(--lightgrey-color);
+    color: var(--black-color);
+    z-index: -2;
+    position: relative;
+}
+
+.case-studies {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 20px 0;
+}
+
+.case-studies-image {
+    font-size: 0; /* to remove white line under picture */
+    flex-basis: 800px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    box-shadow: 0 10px 5px grey;
+}
+
+.case-studies-text {
+    flex-basis: 400px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+}
+
+.squiggle {
+    margin: 20px 0;
+}
+
+.case-studies-1 {
+    flex-direction: row-reverse;
+}
+
+.case-studies-2 {
+
+}
+
+.case-studies-3 {
+    flex-direction: row-reverse;
+}
+
+.view-project-link {
+    text-transform: uppercase;
+    font-weight: bold;
+    text-decoration: none;
+    padding: 50px 0;
+    position: relative;
+}
+
+.view-project-link::after {
+    display: block;
+    content: "";
+    width: 25px;
+    height: 3px;
+    background-color: var(--purple-color);
+    transition: width .35s ease-in-out;
+    position: absolute;
+    left: 150px;
+    bottom: 60px;
+}
+
+.view-project-link:hover::after {
+    width: 50%;
+}
+
+#case-studies-border {
+    padding: 16rem 0 8rem 0;
+}
+
+#case-studies-border::after {
+    content: "";
+    position: absolute;
+    border-top: 0 solid transparent;
+    border-right: 100vw solid white;
+    border-bottom: 70px solid transparent;
+    top: 0;
+    left: 0;
+}
+
+.case-studies-bg {
+    border: 2px solid hotpink;
+    position: relative;
+}
+
+.case-studies-wrapper::before {
+    content: "work";
+    position: absolute;
+    color: white;
+    font-family: 'Roboto', sans-serif;
```

Comment:
Ik zie dat je op veel plekken het font opnieuw zet. Het is makkelijker als je dit op de body zet, dan hoef je het maar één keer te doen namelijk 😉

Reply:
weet niet waarom ik het op meerdere plaatsen heb staan. aangezien het ook in de body staat XD